src/jit/lsraxarch.cpp

```diff
@@ -1013,7 +1013,21 @@ void LinearScan::TreeNodeInfoInitShiftRotate(GenTree* tree)
     if (!shiftBy->isContained())
```

Comment:
Is it possible to be false? I mean `shiftBy->isContained() == true`

Reply:
Yes, `shiftBy` can be contained if it is an immediate, in which case RCX is not used by the instruction.